libc/src/__support/FPUtil/FPBits.h

```diff
@@ -306,188 +304,95 @@ struct FPRepBase : public internal::FPLayout<fp_type> {
     return encode(exp, sig);
   }
 
+  // The floating point number representation as an unsigned integer.
+  StorageType bits = 0;
```

Comment:
[Nit] should this be `bits{}` ? It would make it more obvious that this is also intended to work with class types (e.g. `BigInt`).